db/schema.rb

```diff
@@ -21,4 +21,13 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "transactions", force: :cascade do |t|
+    t.string "Transaction"
+    t.string "credit_card_number"
```

Comment:
I think we should leave out the cc_number column since it is empty. Let's make it a refactor.